packages/core/admin/ee/server/tests/review-workflows.test.api.js

```diff
@@ -242,5 +242,31 @@ describeOnCondition(edition === 'EE')('Review workflows', () => {
         expect(workflowRes.body.data).toBeUndefined();
       }
     });
+    test('It should throw an error if trying to delete all stages in a workflow', async () => {
+      const stagesRes = await requests.admin.put(
+        `/admin/review-workflows/workflows/${testWorkflow.id}/stages`,
+        { body: { data: [] } }
+      );
+      const workflowRes = await requests.admin.get(
+        `/admin/review-workflows/workflows/${testWorkflow.id}?populate=*`
+      );
+
+      if (hasRW) {
+        expect(stagesRes.status).toBe(400);
+        expect(stagesRes.body.error).toBeDefined();
+        expect(stagesRes.body.error.name).toEqual('ApplicationError');
+        expect(stagesRes.body.error.message).toBeDefined();
```

Comment:
I thought about doing this test, but it might be a pain to edit it every time you want to change something or fix the error message.
Note: This would not happen if we were using translation keys, did we already discuss of this point with the engineering team?